backend/src/rgbio.py

```diff
@@ -13,18 +13,26 @@
     © 2022 BeyondRGB
     This code is licensed under the MIT license (see LICENSE.txt for details)
 """
+import sys
+
 import numpy as np
 import rawpy as rp
 from os.path import exists
 from tempfile import TemporaryFile
+from cv2 import imwrite
 
 
-def save_image(img, path):
+def save_image(path, img):
     """ Save image to disk
-    [in] img     : image to save
-    [in] path   : image save location
+    [in] subject :
     """
-    pass
+    out = path + "output.tif"
```

Comment:
it should be .tiff not .tif. Your choice if we append .tiff here or tiff is part of the input for file name